test/functional/page_objects/common_page.js

```diff
@@ -270,15 +270,9 @@ export function CommonPageProvider({ getService, getPageObjects }) {
     }
 
     async getAppNavLinksText() {
```

Comment:
We should probably just remove this and use https://github.com/elastic/kibana/blob/1925ee95664b8f6a9c888f82d0cabd76d33ea6b5/test/functional/services/apps_menu.js#L26

Reply:
thanks for pointing this out, I had no idea this was a thing!

Reply:
It just got added, I stumbled across it on Friday myself.